added welcome page UI

lawrencehsj opened the pull request.

lawrencehsj committed:
added welcome page UI

sinhaugoh commented on lib/core/presentation/const/header.dart:
```diff
@@ -0,0 +1,65 @@
+      ),
+    );
+  }
+} 
```
The way u structure wrong already. Each widget should be in a separate file and widget file shouldn't be store in a const folder.

For example, Dealllogo can be stored in a file called deall_logo,dart.
AppBar can be another widget too.

You dont have to try hard to extract the widgets, the purpose of extraction is mainly so that we can reuse the widget in other place or the widget is complex enough that u think it can be a separate widget. So in this case, if your appbar will only be used in welcome page, then dont extract.

Const folder is to store const values that will not be change throught the app.

lawrencehsj replied:
okay i'll separate the deal logo to another file. the appbar will be used for other pages also eh. 

would storing in core/widgets be ideal?

sinhaugoh replied:
the app logo can be stored in core/presentation/widget

as for the Header class i dont think it is needed, just make ur custom AppBar widget into a stateless or stateful widget

lawrencehsj committed:
update sign in page UI

lawrencehsj committed:
update

-removed header.dart (changed appbar to stl widget)
-added font.dart to const
-added logo.dart to core/presentation/widget

lawrencehsj committed:
added text widget classes

lawrencehsj committed:
added in logo images to images.dart

lawrencehsj committed:
Update welcome_page.dart

lawrencehsj committed:
Update pubspec.yaml
